platform/android/app/src/main/kotlin/coop/polypoly/polypod/PodApi.kt

```diff
@@ -245,15 +246,49 @@ open class PodApi(
         return ValueFactory.newString(info.getVersion())
     }
 
-    private suspend fun handleNetworkHttpPost(args: List<Value>): Value {
-        logger.debug("dispatch() -> network.httpPost")
-        val url = args[0].asStringValue().toString()
-        val contentType = args[1].asStringValue().toString()
-        val body = args[2].asStringValue().toString()
-        val authorization: String? = args[3]?.asStringValue().toString()
-        val error = network.httpPost(url, contentType, body, authorization)
-        return if (error == null) ValueFactory.newNil()
-        else ValueFactory.newString(error)
+    private suspend fun handleEndpointSend(args: List<Value>): Value {
+        logger.debug("dispatch() -> endpoint.send")
+        val endpointId = args[0].asStringValue().toString()
```

Comment:
Is this specified in the Jira ticket?

Reply:
This is just how it's done in postoffice or what do you mean :D

Reply:
This is done exactly like all other api interfaces. The documentation of the api is done in api.ts I'd say